scripts/docker-entrypoint.sh

```diff
@@ -389,6 +389,55 @@ rm "${CATALINA_HOME}"/conf/context.xml.tmpl
 rm "${CATALINA_HOME}"/conf/tomcat-users.xml.tmpl
 rm "${CATALINA_HOME}"/conf/server.xml.tmpl
 
+#
+# Determine whether we are running against a 25.1 + environment as it is necessary to add bc-fips libraries
+# the bootstrap classpath for tomcat.
+#
+if [ -z "${IS_PEGA_25_OR_LATER}" ]; then
+  vc_classpath="/opt/pega/utility/*:/opt/pega/lib/*"
+  vc_javaopts="-Xms${INITIAL_HEAP} -Xmx${MAX_HEAP} ${JAVA_OPTS}"
+  vc_javaopts="${vc_javaopts} --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED"
+  vc_javaopts="${vc_javaopts} --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED"
+  vc_javaopts="${vc_javaopts} --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED"
+  vc_javaopts="${vc_javaopts} --add-opens jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED"
+  vc_javaopts="${vc_javaopts} --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED"
+
+  if [ "${FIPS_140_3_MODE}" == "true" ]; then
+    vc_classpath="/opt/pegabcfips/*:${vc_classpath}"
+    vc_javaopts="${vc_javaopts} -Dorg.bouncycastle.fips.approved_only=true"
+  fi
+
+  krb5_conf="/opt/pega/kerberos/krb5.conf"
+  #
+  # Adding krb5.conf location to JAVA_OPTS
+  #
+  if [ -e "$krb5_conf" ]; then
+    echo "Adding ${krb5_conf} to JAVA_OPTS";
+    vc_javaopts="${vc_javaopts} -Djava.security.krb5.conf=${krb5_conf}"
+  else
+    echo "No krb5.conf was specified in ${krb5_conf}."
+  fi
+
+  echo "java -cp "\"${vc_classpath}\"" ${vc_javaopts} com.pega.cmc.VersionChecker"
+
+  pega_version=$(eval exec java -cp "\"${vc_classpath}\"" "${vc_javaopts}" com.pega.cmc.VersionChecker)
```

Comment:
For a single value like this, you can probably get rid of the `eval exec`.

Reply:
The script is unhappy when I remove it -- will keep it.

Reply:
probably need to also remove one set of quotes each from  `vc_classpath` and `vc_javaopts` to remove it.